src/pages/en/core-concepts/framework-components.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Framework Components
+description: Learn how to use React, Svelte, etc.
+---
+Astro allows you to build your site using JavaScript UI components from a variety of popular frameworks. Astro will run your [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/) components at build time, then render your entire site to static HTML by default, removing all the JavaScript from the final page!
```

Comment:
This is pretty prose-heavy to start. Can we add an example somewhere in this first section to show how easy it is to use a React component in Astro?

Reply:
I hope you like my example!  ;)

Reply:
I love it!